### PR TITLE
Minors on error

### PR DESCRIPTION
**Pull request by cxa**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -35,7 +35,7 @@ const runBsbSync = () => {
 const getBsbErrorMessages = err => err.match(/File [\s\S]*?:\nError: [\s\S]*?\n/g)
 
 const getCompiledFile = (moduleDir, path, callback) => {
-  runBsb((err, res) => {
+  runBsb((err, _, res) => {
```
Comment (rrdelaney) [inline index.js]: Why is this changed?

Comment (cxa) [inline index.js]: As the doc https://nodejs.org/api/child_process.html#child_process_child_process_execfile_file_args_options_callback shows, `callback`'s parameters are `(error, stdout, stderr)`, we only care the `stderr` for emitting error.

Comment (cxa) [inline index.js]: screen shot 2017-06-14 at 00 04 51

As screenshot shows, the `BSB check build spec : OK` is not error.

**Comment by rrdelaney** [comment]
Published as [`v1.5.2`](https://github.com/rrdelaney/bs-loader/releases/tag/v1.5.2)
